### DISCUSSION TITLE
Classes don't apply to headings

### DISCUSSION THREAD

**Discussion by CxdeBits** [tailwindlabs/tailwindcss#8350] [Help] [answered]
Currently I'm using Tailwind in an Angular project. I'm learning Tailwind bit by bit, but there's one thing I'm currently struggling with and I can't seem to resolve what's causing this.

Whenever I'm trying to style a heading or copy some code online not all classes apply to the headings. So my headings never get displayed the right way. For example when I try to make them bold or change font size, that doesn't work. But changing the text color works fine... Unfortunately I can't find information about this issue online.

I've also tried adding this piece of code to the styles.scss file I'm using, but that makes no difference. Any help would be appreciated. Using the latest stable version of Tailwind and Angular at the moment. I have also tried reinstalling Tailwind, no success with that as well... 

```
@layer base {
  h1 {
    @apply font-bold;
    @apply text-green-500;
  }
  h2 {
    @apply font-bold;
    @apply text-green-500;
  }
  h3 {
    @apply font-bold;
    @apply text-green-500;
  }
}
```

```
          <a routerLink="/error">
            <h1 class="font-bold text-green-500">Doesn't work</h1>
            <h2 class="font-bold text-green-500">Doesn't work</h2>
            <h3 class="font-bold text-green-500">Doesn't work</h3>
          </a>
```

![image](https://user-images.githubusercontent.com/17981327/168440153-8f84b891-16c5-4f39-8eed-d8730298bdba.png)

My config file:
```
module.exports = {
  mode: 'jit',
  content: ['./src/**/*.{html,ts}', './src/**/**/*.{html,ts}'],
  darkMode: 'class',
  theme: {
    extend: {}
  },
  plugins: [require('flowbite/plugin'), require('@tailwindcss/typography'), require('@tailwindcss/forms')]
};

```

**Comment by adamwathan** [comment, marked as answer]
Can you share a project that reproduces? It looks like you have other CSS, not just Tailwind.

**Reply by CxdeBits** [reply]
I solved the problems thanks to your reply. There was indeed other CSS that was conflicting with Tailwind. Removed it from my angular.json and now it's working fine.
